### PR TITLE
Fix api change

### PR DESCRIPTION
**Pull request by thimmy687**
This PR should adapt the changes which are made in the main GE project and broke the build

### CODE REVIEW

File: src/GitExtensions.GerritPlugin/GerritUICommands.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ public bool StartCommandLineProcessDialog(IWin32Window owner, IGitCommand comman
             => StartCommandLineProcessDialog(owner, command);
 
         public void StartRemotesDialog() 
-            => gitUICommands.RunCommand(new string[] { "ge.exe", "remotes" });
+            => gitUICommands.RunCommand(new string[] { "remotes" });
```
Comment (thimmy687) [inline src/GitExtensions.GerritPlugin/GerritUICommands.cs:50]: @mast-eu unlikely this is not working

File: src/GitExtensions.GerritPlugin/GerritUICommands.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+﻿using GitCommands;
+using GitUI;
+using GitUIPluginInterfaces;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace GitExtensions.GerritPlugin
+{
+    public sealed class GerritUICommands : GerritUICommandsBase, IGerritUICommands
```
Comment (RussKie) [inline src/GitExtensions.GerritPlugin/GerritUICommands.cs:10]: Is there a reason to create this? Why can't we use `IGitUICommands`?

**Review by RussKie — COMMENTED** [review]
I don't have a strong opinion either way. If it works I'm happy to merge this.

**Comment by mast-eu** [comment]
Should this one be closed, now that #29 is merged?

**Pull request closed without merging**